pkg/http/http.go

```diff
@@ -83,7 +83,7 @@ func RequestToEvent(req *http.Request) (*event.Event, error) {
 	event.Attributes.Extensions = make(map[string]string)
 	for k, _ := range req.Header {
 		if strings.HasPrefix(strings.ToLower(k), "ce-") {
-			event.Attributes.Extensions[k] = req.Header.Get(k)
+			event.Attributes.Extensions[k[len("ce-"):]] = req.Header.Get(k)
```

Comment:
Did you want to make it into a constant?

Reply:
what happens when the header's name is `ce-` ?  Seems like an error condition and it should ignore the header for the purposes of saving CE extensions.

Reply:
I am trying to not over engineer this tool

Reply:
There is not a ton of validation in this tool. I think it should be fairly barebones.